File: beginner/1012.py
A, B, C = map(float, input().split())
TRIANGULO = (A * C) / 2
CIRCULO = (C ** 2) * 3.14159
TRAPEZIO = ((A + B) * C) / 2
QUADRADO = B ** 2
RETANGULO = A * B
print('TRIANGULO: {:.3f}'.format(TRIANGULO))
print('CIRCULO: {:.3f}'.format(CIRCULO))
print('TRAPEZIO: {:.3f}'.format(TRAPEZIO))
print('QUADRADO: {:.3f}'.format(QUADRADO))
print('RETANGULO: {:.3f}'.format(RETANGULO))

File: beginner/1012.cpp
#include <iostream>
#include <iomanip>

using namespace std;

int main() {
  const double pi = 3.14159;
  double a, b, c;
  cin >> a >> b >> c;
  cout << fixed << setprecision(3);
  cout << "TRIANGULO: " << (a * c) / 2 << endl;
  cout << "CIRCULO: " << pi * (c * c) << endl;
  cout << "TRAPEZIO: " << ((a + b) * c) / 2 << endl;
  cout << "QUADRADO: " << b * b << endl;
  cout << "RETANGULO: " << a * b << endl;

  return 0;
}
